lib/instruments/supertest.js

```diff
@@ -54,8 +68,12 @@ function requestHeaders(superTest) {
 
 function responseHeaders(superTest, response) {
   // only show response headers that were asserted on
-  var names = Object.keys(superTest._fields).map(function(s) { return s.toLowerCase() });
-  return _.pick(response.headers, function(value, key) {
-    return names.indexOf(key.toLowerCase()) !== -1;
-  });
+  if (superTest._fields) {
+    var names = superTest._fields.map(function(s) { return s.toLowerCase() });
+    return _.pick(response.headers, function(value, key) {
+      return names.indexOf(key.toLowerCase()) !== -1;
+    });
+  } else {
+    return {};
+  }
```

Comment:
It looks like the other change (`assert` -> `expect`) is backwards compatible because we still shim both functions. What about this change to `responseHeaders`?